src/Statistics.jl

```diff
@@ -1035,8 +1035,10 @@ end
     @assert n > 0 # this case should never happen here
 
     m = alpha + p * (one(alpha) - alpha - beta)
-    aleph = n*p + oftype(p, m)
-    j = clamp(trunc(Int, aleph), 1, n-1)
+    # Using muladd here avoids some rounding errors when aleph is an integer
+    # The use of oftype supresses the promotion caused by alpha and beta
+    aleph = muladd(n, p, oftype(p, m))
```

Comment:
AFAIK `muladd` only aims at performance, it's `fma` which aims at precision (even if in practice both are often the same I think). Performance isn't an issue here so we're fine.
```suggestion
    # Using fma here avoids some rounding errors when aleph is an integer
    # The use of oftype supresses the promotion caused by alpha and beta
    aleph = fma(n, p, oftype(p, m))
```